clinvar_ingest/api/middleware.py

```diff
@@ -0,0 +1,25 @@
+import logging
+import time
+import uuid
+from typing import Callable
+
+from starlette.middleware.base import BaseHTTPMiddleware, Request, Response
+
+from clinvar_ingest.api.constants import MS_PER_S
+
+logger = logging.getLogger("api")
+
+
+class LogRequests(BaseHTTPMiddleware):
+    async def dispatch(self, request: Request, call_next: Callable) -> Response:
```

Comment:
I threw this logger in so we'd have local logs for testing... but CloudRun has logging that's doing basically the same thing 🤷